benchmarking/commons/benchmark_definitions/lcbench.py

```diff
@@ -26,6 +26,11 @@ def lcbench_benchmark(dataset_name, datasets=None):
         dataset_name=dataset_name,
         surrogate="KNeighborsRegressor",  # 1-nn surrogate
         surrogate_kwargs={"n_neighbors": 1},
+        # surrogate="RandomForestRegressor",
+        # add_surrogate_kwargs={
+        #     "predict_curves": True,
+        #     "fit_differences": ["time"],
+        # },
```

Comment:
might want to remove if no longer needed

Reply:
I'll move it into the comment